pkg/store/copr/batch_coprocessor.go

```diff
@@ -819,12 +822,24 @@ func filterAllStoresAccordingToTiFlashReplicaRead(allStores []uint64, aliveStore
 	return
 }
 
+func getAllUsedTiflashStores(allTiFlashStores []*tikv.Store, allUsedTiflashStoresMap map[uint64]struct{}) []*tikv.Store {
```

Comment:
What happens if a store exists in `allUsedTiFlashStoresMap` not not exist in `allTiFlashStores`?